### PR TITLE
Fix for #86

### PR DESCRIPTION
**Pull request by kikoanis**

### CODE REVIEW

**Comment by m-allanson** [comment]
@KyleAMathews It'd be great to see this merged in to fix #86.

File: src/index.js [hunk 1]
```diff
@@ -154,7 +153,21 @@ export default class Headroom extends Component {
...
+    delete divProps.onUnfix
+    delete divProps.disableInlineStyles
+    delete divProps.disable
+    delete divProps.parent
+    delete divProps.children
+    delete divProps.upTolerance
+    delete divProps.downTolerance
+    delete divProps.pinStart
+
+    const { style, wrapperStyle, ...rest } = divProps
```
Comment (m-allanson) [inline src/index.js]: I was wondering... could you shorten this by doing something like this?

e.g. (untested):

```
    const { style, wrapperStyle, ...rest } = this.props
    const redundantKeys = Object.keys(this.propTypes)
    redundantKeys.forEach((key) => {
        delete rest[key]
    })
```

**Comment by KyleAMathews** [comment]
Yeah, I will soon as I have some more open source time...

**Comment by m-allanson** [comment]
Great, thanks!

File: src/index.js (hunk 1 above)
Comment (KyleAMathews) [inline src/index.js]: I like this. I'll merge this PR and would you like to make a PR for your change after that? That's mostly if you want the Git attribution. If you don't care much I'll just copy in your change myself.

Comment (m-allanson) [inline src/index.js]: @KyleAMathews Nice! I'm happy for you to copy the change in.

**Pull request merged**